### DISCUSSION TITLE
What is the difference between val/mAP_50 and val/bbox_mAP_50?

### DISCUSSION THREAD

**Discussion by sarmientoj24** [open-mmlab/mmdetection#9168] [Q&A] [answered]
They seem different. And what is the difference of bbox and proposal in evaluation?

**Comment by jbwang1997** [comment]
`bbox` indicates the outputs of the model are detection results and the code need to calculate the AP and AR for detection. `proposal` indicates the outputs of the model are proposals from RPN (which is only a list of np.ndarray without classes) and the code need to calculate the AR.

**Comment by jbwang1997** [comment]
bbox_mAP_50 indicates the mAP for bboxes especially. There are also segm_mAP_50 for instance segmentation.

**Comment by sarmientoj24** [comment]
what do you usually use for the publications?

**Comment by sarmientoj24** [comment]
between mAP_50 and bbox_mAP_50 and why?

**Comment by JackMa-coder** [comment]
the result of bbox is two stage's performance, while that of proposal is RPN's performance.

---Original---
From: "James-Andrew R. ***@***.***&gt;
Date: Wed, Jun 29, 2022 16:14 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [open-mmlab/mmdetection] What is the difference between val/mAP_50and val/bbox_mAP_50? (Issue #8280)




 
They seem different. And what is the difference of bbox and proposal in evaluation?

**Comment by ZwwWayne** [comment, marked as answer]
For publications, we usually use box mAP, which is more strict than map_50(AP at iou 0.5).
